pilot/pkg/networking/core/v1alpha3/dns_listener.go

```diff
@@ -73,7 +73,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarDNSListener(node *model.Proxy,
 		ClientConfig: &dnsfilter.DnsFilterConfig_ClientContextConfig{
 			ResolverTimeout: ptypes.DurationProto(resolverTimeout),
 			// no upstream resolves. Envoy will use the ambient ones
-			MaxPendingLookups: 64, // arbitrary
+			MaxPendingLookups: 128, // arbitrary
```

Comment:
We can probably go up to 256 if we want -that is what Envoy docs and integration tests are defaulting it to.